ansible/files/goss/goss.service

```diff
@@ -7,6 +7,7 @@ ExecStart=/usr/bin/goss -g /etc/goss/health.yml serve &
 ExecStop=/bin/kill -WINCH ${MAINPID}
 KillSignal=SIGKILL
 User=aeternity
+StandardOutput=null
```

Comment:
what about init.d ? Should we care about it at all?

Reply:
It's not used since there is a service file. Also does not log to syslog and not managed by rsyslogd